### ISSUE TITLE
Editable nodes

### ISSUE THREAD

**Issue by MAccadia** [dundalek/atom-markdown-mindmap#7]
Hi,

I find tree view of documents interesting by it self but it would be great to be able to do more editing like drag and drop to rearrange sections and title editing.

I've seen [this app](https://gingkoapp.com/) which I find inspiring but to me there is to much things on screen ; I like distraction free writing.

Best,

M.A.

**Comment by dundalek** [comment]
Yup, this would be nice. I want to explore bit more of the visualization side first, so this feature is not on my list at the moment.
